package.json

```diff
@@ -62,8 +65,15 @@
     "@canonical/cookie-policy": "3.2.0",
     "@canonical/global-nav": "2.4.5",
     "@canonical/latest-news": "1.2.0",
+    "@canonical/react-components": "^0.17.1",
     "@reduxjs/toolkit": "^1.5.0",
+    "@stripe/react-stripe-js": "^1.4.1",
+    "@stripe/stripe-js": "^1.15.0",
     "date-fns": "2.17.0",
+    "formik": "^2.2.8",
```

Comment:
While we're using formik on the two app UI's right now we're evaluating alternatives that don't have the same performance and api restrictions.... Just something to keep in mind for later :)

Reply:
I did try to use react-hook-form in the first place but the vanilla components weren't playing nice with it and so I turned to Formik instead

Reply:
I'm interested to hear about that issue as that one is currently on the top of the list :)

Reply:
IIRC the issue was that vanilla react component don't forward the ref to the actual input field and so it wasn't possible to control it (reset / set to a specific value)
https://react-hook-form.com/get-started#Integratinganexistingform